src/Tribe/Admin/Settings/Service_Provider.php

```diff
@@ -33,7 +33,7 @@ protected function hooks() {
 			return;
 		}
 
-		add_action( 'tribe_settings_before_content_tab_event-tickets', [ $this, 'render_settings_banner' ] );
+//		add_action( 'admin_notices', [ $this, 'render_settings_banner' ], 99 );
```

Comment:
If we can't make use of `tribe_notice` does it make more sense of keeping the code within this class and making use of `admin_notices` ?

Also, we can control the position of notice by using higher priority here which is not possible with `tribe_notice`